02Lesson/01Project/index.html

```diff
@@ -1,9 +1,34 @@
 <!DOCTYPE html>
-<html>
-<head>
-  <title></title>
-</head>
-<body>
-
-</body>
+<html style="background-color: powderblue">
+  <head>
+    <title>Motorcycle Taxidermy</title>
+  </head>
+  <body>
+    <center><img src="https://cyclingdynamics.files.wordpress.com/2013/10/mad-cow-motorcycle_wasav_38681.jpg" alt="MadCow Motorcycle. Skeletal Choppers made from real bones!"></center>
```

Comment:
The `center` element is obsolete in HTML5. Check W3C validator when saving files in atom.